CONTRIBUTORS.txt

```diff
@@ -36,20 +43,23 @@ needs to be listed here.
 - Daniel Dunbar <daniel_dunbar@apple.com>
 - Dario Rexin <drexin@apple.com>
 - David Evans <d.evans@apple.com>
+- David Nadoba <d_nadoba@apple.com>
 - David Skrundz <david.skrundz@gmail.com>
+- Donghyeon Kim <wplong11@users.noreply.github.com>
 - Doug Friedman <dfriedm3@gmail.com>
 - Emil Marashliev <marashliev@gmail.com>
 - Esteban Torres <me@estebantorr.es>
 - Eugen <eugenf78@gmail.com>
 - Fabian Fett <fabianfett@apple.com>
 - Frank Kair <frankkair@gmail.com>
-- Franz Busch <privat@franz-busch.de>
+- Franz Busch <f.busch@apple.com>
 - Gautier Delorme <gautier.delorme@gmail.com>
 - George Barnett <gbarnett@apple.com>
 - Gopal Sharma <gopal@the-sharmas.org>
 - Graeme Jenkinson <graeme_jenkinson@apple.com>
 - Gregor Milos (Grzegorz Miłoś) <gmilos@apple.com>
 - GuangGuang <lihansey163@163.com>
+- Guillaume Lessard <glessard@users.noreply.github.com>
```

Comment:
@glessard  if you would like to have a real email address here, please add a comment with your preferred address. Thank you!

Reply:
```suggestion
- Guillaume Lessard <guillaume.lessard@apple.com>
```